libcodechecker/libauth/cc_ldap.py

```diff
@@ -273,6 +274,11 @@ def auth_user(ldap_config, username=None, credentials=None):
     """
     Authenticate a user.
     """
+    if not username:
```

Comment:
Should an error be printed here? Either way, I think for this we could express it shorter:

```python
if not username or not credentials:
```